doc/release-notes/iceoryx2-unreleased.md

```diff
@@ -122,6 +122,9 @@
     please refer to `config/README.md` on how to generate a default config file.
   [#831](https://github.com/eclipse-iceoryx/iceoryx2/issues/831)
 
+* Recreate the cmake patch, used in the yocto layer
+  [#663](https://github.com/eclipse-iceoryx/iceoryx2/issues/663)
+
```

Comment:
I think this can be moved to the features section, e.g. line 20. It introduces a new option which is ON by default, so does not break existing builds.

I think when it is moved to the other section, the markdown linter will also stop complaining.

Reply:
Okay, no problem. We just have to merge this PR quickly :)